lib/cash-in/cash-in-tx.js

```diff
@@ -35,12 +36,14 @@ function post (machineTx, pi) {
       let blacklisted = false
       let addressReuse = false
       let walletScore = {}
+      let testingAddress = false
 
-      return Promise.all([settingsLoader.loadLatest(), checkForBlacklisted(updatedTx), doesTxReuseAddress(updatedTx), getWalletScore(updatedTx, pi)])
-        .then(([{ config }, blacklistItems, isReusedAddress, fetchedWalletScore]) => {
+      return Promise.all([settingsLoader.loadLatest(), checkForBlacklisted(updatedTx), doesTxReuseAddress(updatedTx), getWalletScore(updatedTx, pi), checkForTestingAddress(updatedTx)])
+        .then(([{ config }, blacklistItems, isReusedAddress, fetchedWalletScore, isAddressTesting]) => {
```

Comment:
Another nitpicky thing, but the variable name makes more sense this way imo

```suggestion
        .then(([{ config }, blacklistItems, isReusedAddress, fetchedWalletScore, isTestingAddress]) => {
```